libs/pika/algorithms/tests/performance/foreach_scaling.cpp

```diff
@@ -234,7 +234,7 @@ int pika_main(pika::program_options::variables_map& vm)
     chunk_size = vm["chunk_size"].as<int>();
     num_overlapping_loops = vm["overlapping_loops"].as<int>();
     disable_stealing = vm.count("disable_stealing");
-    fast_idle_mode = vm.count("fast_idle_mode");
+    scheduler_mode::fast_idle_mode = vm.count("fast_idle_mode");
```

Comment:
This is an unfortunately named `bool`. It should not be changed (this should fail to compile anyway).

Reply:
Same further down in the same file.